lib/dndmonster.sty

```diff
@@ -17,7 +17,7 @@
 % e.g. \dice{2d6+3} prints "10 (2d6 + 3)"
 \newcommand{\dice}[1]{%
   \StrSubstitute{#1}{ }{}[\DiceArg]%				strip whitespaces
-  \StrCut{\DiceArg}{d}\DiceNum\DiceSides%			split string
```

Comment:
For the time being, lets keep the literal "d" here. The regex version will be able to accept any 1 letter, but a user who recompiles for different languages does not want to have to keep changing the letter in the input.

Reply:
Recompiling for different languages doesn't really work anyway though because of `name`, `targets`, `ordmgwhen` and `extra`. Since these will have values in the chosen language, I think it would make sense to use the `\dname` corresponding to the same language (from the user perspective).

Since the regex will be liberal in that regard, I'm okay with keeping the `d` for now.